OrbitGl/TimeGraph.cpp

```diff
@@ -250,7 +250,7 @@ void TimeGraph::ProcessTimer(const TimerInfo& timer_info, const FunctionInfo* fu
     capture_max_timestamp_ = timer_info.end();
   }
 
-  if (function != nullptr && function->orbit_type() != FunctionInfo::kNone) {
+  if (function != nullptr && function->orbit_type() != FunctionInfo::kNone && timer_info.type() == TimerInfo::kNone) {
```

Comment:
As discussed online, better if we have a comment for this "cryptic" condition

Reply:
Done, good suggestion!